src/components/Dissolution/CompleteResolution.vue

```diff
@@ -443,96 +442,87 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin) {
     ]
   }
 
-  private validateIsBetweenDates (minDate: Date, maxDate: Date, dateToValidate: Date): boolean {
-    if (!dateToValidate) { return true }
-    if (dateToValidate >= minDate && dateToValidate <= maxDate) { return true }
-    return false
-  }
-
   /** The minimum date that can be entered (can't be earlier than incorporation date ). */
-  private get resolutionDateMinStr (): string {
+  get resolutionDateMinStr (): string {
     return this.dateToYyyyMmDd(this.foundingDate)
   }
 
   /** The minimum date that can be entered (can't be earlier than incorporation date ). */
-  private get resolutionDateMin (): Date {
+  get resolutionDateMin (): Date {
     return this.foundingDate
   }
 
   /** The maximum date that can be entered (today). */
-  private get resolutionDateMaxStr (): string {
-    const date = new Date()
-    return this.dateToYyyyMmDd(date)
+  get resolutionDateMaxStr (): string {
+    return this.dateToYyyyMmDd(this.getCurrentJsDate)
```

Comment:
Alternately, this could use `this.getCurrentDate`, but the code as written is consistent with similar getters here so I think this is better.